### PR TITLE
embrace-symbol-upload 1.0.0

### PR DESCRIPTION
**Pull request by chawco**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/embrace-symbol-upload/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  Upload iOS debug symbols to Embrace through Bitrise CI integration
+description: |
+  Upload iOS debug symbols to Embrace through Bitrise CI. This will make your integration with Embrace more efficient. Environment Variables are supported via the upload tool.
+
+  Env Vars:
+  ---------
+  EMBRACE_APP: Your Embrace app idenifier
+  EMBRACE_API_TOKEN: Your Embrace upload API token
+website: https://github.com/embrace-io/bitrise-step-embrace-symbol-upload
+source_code_url: https://github.com/embrace-io/bitrise-step-embrace-symbol-upload
```
Comment (ofalvai) [inline steps/embrace-symbol-upload/1.0.0/step.yml:12]: This is 404 for me, it should be a public repo

File: steps/embrace-symbol-upload/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+support_url: https://github.com/embrace-io/bitrise-step-embrace-symbol-upload/issues
+published_at: 2023-10-05T14:49:30.418918-04:00
+source:
+  git: https://github.com/embrace-io/bitrise-symbol-upload.git
+  commit: f24fc1c9e96311333c2deab2ae69f3ad4b13409b
+project_type_tags:
+- ios
```
Comment (ofalvai) [inline steps/embrace-symbol-upload/1.0.0/step.yml:19]: I think this step is relevant for cross-platform technologies too (when the project is targeting iOS), we recommend listing the relevant cross-platform types from here: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#platforms

File: steps/embrace-symbol-upload/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+type_tags:
+- artifact-info
+toolkit:
+  bash:
+    entry_file: step.sh
```
Comment (ofalvai) [inline steps/embrace-symbol-upload/1.0.0/step.yml:24]: I had a look at what this script does, and it's not clear for me where the CLI binary is coming from. Is it expected that the users of the step perform some other thing first? If so, it's probably better to highlight in the step description and handle the missing binary gracefully with a nice error message.

**Review by ofalvai — CHANGES_REQUESTED** [review]
Hey @chawco, I flagged some minor problems with the step, please take a look

File: steps/embrace-symbol-upload/1.0.0/step.yml (hunk 3 above)
Comment (chawco) [reply steps/embrace-symbol-upload/1.0.0/step.yml:24]: At the moment the binaries are checked in with the step definition. Currently the tool is only shipped as part of our Cocoapod, but we're in the process of defining more distribution mechanisms, expanding the platforms supported by the CLI, and adding more capabilities to it in general. Eventually the intent is to add a separate step which installs the CLI.

File: steps/embrace-symbol-upload/1.0.0/step.yml (hunk 1 above)
Comment (chawco) [reply steps/embrace-symbol-upload/1.0.0/step.yml]: These should be updated properly now.

File: steps/embrace-symbol-upload/1.0.0/step.yml (hunk 2 above)
Comment (ofalvai) [reply steps/embrace-symbol-upload/1.0.0/step.yml:19]: Got it, thank you!

File: steps/embrace-symbol-upload/1.0.0/step.yml (hunk 3 above)
Comment (ofalvai) [reply steps/embrace-symbol-upload/1.0.0/step.yml:24]: Makes sense, thank you

**Review by ofalvai — APPROVED** [review]
Looks good now, thank you for making the changes @chawco. And welcome to the Bitrise step library 🚀

**Pull request merged**